source/default/_patterns/00-protons/legacy/css/components/UTC-custom-blocks/_utc_blockquotes.css

```diff
@@ -1,30 +1,42 @@
-
-
 blockquote {
-  /* font-family: $utc-headings; */
-  font-weight: 700;
-  /* //background-color: $utc-lightyellow; */
-  border-left: 12px solid #CBD5E0;
-  /* margin: 1.5em 9px; */
-  padding: 0.5em 9px;
-  quotes: "\201C""\201D""\2018""\2019";
+    width: 100%;
+    margin: 2rem auto;
+    font-style: italic;
+    padding: 2rem 3rem;
+    background: #eee;
```

Comment:
We probably talk about this next week. We need to choose colors that are part of color pallete.
The @apply statement lets you use predefined tailwindcss colors or we can justify modify or adding a new one.